### PR TITLE
Describe how to add a product with configuration for the Managing a Shopping Cart page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request describes how to add a product with configuration for the Managing a Shopping Cart page. Removes the image for the Configurable product, because you can add 4 types of products with configurations to the cart.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/sales/shopping-assisted-cart-manage.html?itm_source=merchdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=manage%20shoping

### CODE REVIEW

File: src/sales/shopping-assisted-cart-manage.md [hunk 1]
```diff
@@ -63,17 +63,16 @@ _Shopping Cart in in the customer account_
    ![]({% link images/images-ee/customer-account-manage-cart-update-cart.png %}){: .zoom}
    _Cart Updated_
 
-### Add a configurable product
+### Add a product with configuration
+
+There are 4 types of products that need to be configured before adding to the cart: `Bundle Product`, `Configurable Product`, `Grouped Product`.
```
Comment (leslietilling) [inline src/sales/shopping-assisted-cart-manage.md:68]: We spell out numbers under 10 as a general tech writing style rule and there are only three named here

```suggestion
There are three types of products that need to be configured before adding to the cart: `Bundle Product`, `Configurable Product`, and `Grouped Product`.
```

File: src/sales/shopping-assisted-cart-manage.md [hunk 2]
```diff
@@ -154,7 +153,7 @@ Items can be transferred to the cart from a customer wish list, as well as recen
 
 ## Create the order
 
-1. Click <span class="btn">Create Order<span/>.
+1. Click <span class="btn">Create Order</span>.
```
Comment (leslietilling) [inline src/sales/shopping-assisted-cart-manage.md:156]: Good fix!! thanks :)

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just a minor tweak

File: src/sales/shopping-assisted-cart-manage.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/sales/shopping-assisted-cart-manage.md]: Thanks, done ✅

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**